clients/web/src/collections/UserCollection.js

```diff
@@ -8,6 +9,15 @@ var UserCollection = Collection.extend({
     // Override default sort field
     sortField: 'lastName',
     secondarySortField: 'firstName'
+}, {
+    getTotalCount: function () {
+        return restRequest({
+            path: 'user/details'
```

Comment:
In light of https://github.com/girder/girder/pull/2273 , can you make this parameter be `url: ` instead of `path: `?